src/scss/_elements.layout.scss

```diff
@@ -3,4 +3,5 @@ body {
   background-color: $black;
   color: $white;
   font-family: $font-body;
+  overflow-x: hidden; // Prevent horizontal scroll in Firefox, Edge
```

Comment:
This prevents horizontal scroll but cuts text long words within `<h2>`'s. If we add `overflow-wrap: break-word;` the word will wrap. Unfortunately it does not show/add a hyphen when wrapping the word so long words may be hard to decipher.

<img width="653" alt="firefox" src="https://user-images.githubusercontent.com/12678977/60991635-e5b68f00-a318-11e9-83e7-544ede660000.png">

Reply:
I did a little bit of reading and am wondering if Firefox wants the `lang` attribute set to `en` in order for hyphens to work. 🤔I don't know if that will solve it or not.